libs/design-system/tune-menu/src/lib/tune-menu.component.html

```diff
@@ -0,0 +1,91 @@
+@if (!menuOptions().hideHeaderSection) {
+  <div hraFeature="filters" class="header-section">
+    <div class="drawer-header">
+      <span class="drawer-title">{{ menuOptions().headline }}</span>
+      <div class="filler"></div>
+      @if (!isWideScreen() || !menuOptions().hideCloseButton) {
+        <button
+          hraFeature="toggle"
+          [hraClickEvent]="{ opened: formClosed() }"
+          hraHoverEvent
+          mat-icon-button
+          (click)="toggleForm.emit()"
+          aria-label="Open or close tune menu"
+        >
+          <hra-icon class="menu-icon" fontIcon="close" />
+        </button>
+      }
+    </div>
+    <div class="header-subtext">{{ menuOptions().subtext }}</div>
+  </div>
+}
+<ng-scrollbar hraScrollOverflowFade>
+  <form>
+    @if (!menuOptions().hideCustomizeSection) {
```

Comment:
Replace these controls with ng-content projection. I think you can use `contentChildren('*')` to detect whether you have controls or not. Or you can add a directive that that has to be applied to projected elements. Or possibly even use https://material.angular.dev/cdk/observers/overview